### ISSUE TITLE
About the Calculation of Pseudo Label Loss Function

### ISSUE THREAD

**Issue by willemeng** [ccc870206/3D-PL#5]
When I read your paper, I found a problem about a pseudo-label loss function,
![image](https://user-images.githubusercontent.com/72070321/220900985-89371344-9769-4530-a02a-c958b5a5bea0.png)
![image](https://user-images.githubusercontent.com/72070321/220901078-fbbb6737-9845-4907-a30c-a028f196a9d3.png)
according to Figure 1, will the formula in Figure 2 always be equal to 0, because Mconsist* Yr = Ycons.I'm stumped on this and would be very grateful if you could give me some guidance.
When will the complete code be uploaded?

**Comment by wasidennis** [comment]
Thanks for the great question. Our pseudo-labels are pre-calculated based on the first-stage pre-trained model and then fixed during the second-stage training (i.e., y_cons is fixed in Eq 4 based on the very initial y_r). Afterward, y_r will be changing, so the loss value would not be 0.
